tests/integration-tests/rollout_test.go

```diff
@@ -55,10 +55,11 @@ func TestArgoRolloutWork(t *testing.T) {
 		name string
 		app  string
 	}{
-		{
-			name: "it can sync manifests into the cluster",
-			app:  "rollout-" + appSuffix,
-		},
+		// TODO this will be fixed in Ref SRX-PA568W
```

Comment:
Why is this commented? And if it is failing then isn't this an issue?

Reply:
There is *an* issue, but i don't think it's too critical.
The problem is that we have no way yet to wait for the manifest-export-service to be done. That's why the result is not reaching argoCd.
To solve this we need to:
* get the export-service to run in the integration-test
* find a way to wait for the export-service to be done - currently it has no endpoints at all.